### PR TITLE
ENH: clasp support; GAS support

### LINKED ISSUE #8: Project should be based on Google Add-Ons platform

### LINKED ISSUE #9: Programmatically provision Google Scripts

### PR DESCRIPTION
**Pull request by thermokarst**
Fixes #8
Fixes #9 

---

Pair-programmed with @turan1393

### CODE REVIEW

**Comment by thermokarst** [comment]
@turan1393, want to review this PR? See if you can pull down these changes and test it out locally. We can do that together on Friday if you wish, just let me know!

**Review by turanoo — APPROVED** [review]
This looks good to me! I was able to use it on my machine.

File: README.md [hunk 1]
```diff
@@ -23,3 +23,12 @@ yarn install
...
 npm run build
 ```
+
+### Deploy Instructions
+
+```bash
+# You can also install clasp globally and login that way, too - the token is
+# shared at the user level.
+./node_modules/.bin/clasp login
+cd dist && ./..node_modules/.bin/clasp push && cd ..
```
Comment (turanoo) [inline README.md]: Found a typo in the deploy instructions.
`cd dist && ./..node_modules/.bin/clasp push && cd ..` is supposed to be `cd dist && ./../node_modules/.bin/clasp push && cd ..`

**Review by turanoo — CHANGES_REQUESTED** [review]
Please see the bug I found!

**Pull request merged**